Assets/Scripts/Interaction/DrawingTest.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Mirror;
+using UnityEngine;
+
+namespace Player
+{
+    //TODO: Should become part of the interaction system, once that is implemented in a more robust manner
+    public class DrawingTest : NetworkBehaviour
+    {
+        [SerializeField] private GameObject crayonDecal;
+        [SerializeField] private float drawingTickRate = 0.01f;
+
+        private float currentTick = 0f;
+        private Camera mainCamera;
+
+        private void Start()
+        {
+            mainCamera = Camera.main;
+        }
+
+        private void Update()
+        {
+            TrackDrawingTick();
+            HandleDraw();
+        }
+
+        private void TrackDrawingTick()
+        {
+            currentTick += Time.deltaTime;
+            if (currentTick > drawingTickRate)
+            {
+                currentTick = 0f;
+            }
+        }
+        
+        private void HandleDraw()
+        {
+            if (!isLocalPlayer)
+            {
+                return;
+            }
+            
+            if (!Input.GetButton("Draw"))
+            {
+                return;
+            }
+
+            if (currentTick > 0f)
+            {
+                return;
+            }
```

Comment:
For just simple return statements like this, I actually prefer going against the guidelines and having it on one line. This is barely readable to me